packages/jest-cli/src/cli/index.ts

```diff
@@ -36,7 +36,7 @@ export async function run(maybeArgv?: Array<string>, project?: Config.Path) {
   } catch (error) {
     clearLine(process.stderr);
     clearLine(process.stdout);
-    console.error(chalk.red(error.stack));
+    console.error(chalk.red(error));
```

Comment:
Can anyone think of a reason why this should log `error.stack`?

Reply:
I guess @aaronabramov had some valid reason while adding this, but also may have been a leftover from a refactor. I'm onboard with the change

Reply:
Maybe
```js
if(error.stack) console.error(`Stack:\n${error.stack}`)
```

Reply:
Changed it to an if-else that looks okay both with and without deleting the stack